modules/apps/document-library-opener/document-library-opener-google-drive-test/src/testFunctional/GoogleDrive.testcase

```diff
@@ -424,6 +424,20 @@ definition {
 
 		Navigator.gotoPage(pageName = "Documents and Media Page");
 
+		DMNavigator.gotoEditDocumentPG(dmDocumentTitle = "Untitled");
+
+		AssertTextEquals(
```

Comment:
@Jeremy-Ch there are a lot of redundant assertions here. I would rather there be a macro for this that can cover multiple cases so we don't need to be using functions at the test case level. Can you file a test improvement ticket for this?